scripts/1-validate-packages.sh

```diff
@@ -10,7 +10,6 @@ validate () {
   query=$1;
   schema=$2;
   for file in $(find $PKG_DIR -name $query); do
-    echo "Validating $file";
```

Comment:
Eventually these bash scripts should be replaced with more portable python.  Don't want to get too fancy here.